Packs/PrismaCloud/Playbooks/playbook-Prisma_Cloud_-_Network__API_and_Anomaly_Incidents.yml

```diff
@@ -0,0 +1,1999 @@
+  required: false
+  description: The ServiceNow ticket type. Options are "incident", "problem", "change_request", "sc_request", "sc_task", or "sc_req_item". Default is "incident".
+  playbookInputQuery:
+- key: serviceNowCategory
+  value: {}
+  required: false
+  description: The category of the ServiceNow ticket.
+  playbookInputQuery:
+- key: serviceNowAssignmentGroup
+  value: {}
+  required: false
+  description: The group to which to assign the new ticket.
+  playbookInputQuery:
+- key: ZendeskPriority
+  value: {}
+  required: false
+  description: The urgency with which the ticket should be addressed. Allowed values are "urgent", "high", "normal", or "low".
+  playbookInputQuery:
+- key: ZendeskRequester
+  value: {}
+  required: false
+  description: The user who requested this ticket.
+  playbookInputQuery:
+- key: ZendeskStatus
+  value: {}
+  required: false
+  description: The state of the ticket. Allowed values are "new", "open", "pending", "hold", "solved", or "closed".
+  playbookInputQuery:
+- key: ZendeskSubject
+  value: {}
+  required: false
+  description: The value of the subject field for this ticket.
+  playbookInputQuery:
+- key: ZendeskTags
+  value: {}
+  required: false
+  description: The array of tags applied to this ticket.
+  playbookInputQuery:
+- key: ZendeskType
+  value: {}
+  required: false
+  description: The type of this ticket. Allowed values are "problem", "incident", "question", or "task".
+  playbookInputQuery:
+- key: ZendeskAssigne
+  value: {}
+  required: false
+  description: The agent currently assigned to the ticket.
+  playbookInputQuery:
+- key: ZendeskCollaborators
+  value: {}
+  required: false
+  description: The users currently CC'ed on the ticket.
+  playbookInputQuery:
+- key: ZenDeskDescription
+  value: {}
+  required: false
+  description: The ticket description.
+  playbookInputQuery:
+- key: CreateTicket
+  value:
+    simple: "False"
+  required: false
+  description: Whether to create a ticket in ZenDesk or ServiceNow. Insert True or False
+  playbookInputQuery:
+- key: StopForRecommendations
+  value:
+    simple: "True"
+  required: false
+  description: ""
+  playbookInputQuery:
+- key: EarlyContainment
+  value:
+    simple: "True"
+  required: false
+  description: Whether to perform early containment.
+  playbookInputQuery:
+- key: CloudResponse
+  value:
+    simple: "False"
+  required: false
+  description: Whether to perform cloud response.
+  playbookInputQuery:
+inputSections:
+- inputs:
+  - serviceNowShortDescription
+  - serviceNowImpact
+  - serviceNowUrgency
+  - serviceNowSeverity
+  - serviceNowTicketType
+  - serviceNowCategory
+  - serviceNowAssignmentGroup
+  name: Incident Management - ServiceNow
+  description: ""
+- inputs:
+  - ZendeskPriority
+  - ZendeskRequester
+  - ZendeskStatus
+  - ZendeskSubject
+  - ZendeskTags
+  - ZendeskType
+  - ZendeskAssigne
+  - ZendeskCollaborators
+  - ZenDeskDescription
+  name: Incident Management - ZenDesk
```

Comment:
Please update the input group name to match the input group name in the core playbooks.